manageiq-operator/config/manager/manager.yaml

```diff
@@ -12,6 +12,15 @@ spec:
       labels:
         name: manageiq-operator
     spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: kubernetes.io/arch
+                operator: In
+                values:
+                - amd64
```

Comment:
Do we also want `s390x` and `ppc64le` like in downstream?

Reply:
In downstream we have a separate setNodeAffinity function that adds the other arches to the Kafka CR

Reply:
> Do we also want s390x and ppc64le like in downstream?

No, we only build x86_64 upstream

>  I'm not sure how this will work in downstream though if we have additional arches.

We don't use this file downstream.  The downstream operator will deploy the manageiq operator with the correct values.